boards/arm/decawave_dwm1001_dev/decawave_dwm1001_dev.dts

```diff
@@ -43,6 +43,26 @@
 		};
 	};
 
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0_red: pwm_led_0 {
+			pwms = <&pwm0 14>;
```

Comment:
In this case, I'm not sure if all these LEDs should be made PWM ones. Assigning of the pins to the PWM peripheral prevents them from being controlled as GPIOs when the PWM peripheral is enabled. Perhaps it would be safer to keep only pin 22 assigned to `pwm0` (as it was so far) and to define only the LED it drives ("Red LED 2") as a PWM one? But I don't really know this board, it's not a Nordic one (it only uses a Nordic chip), so my concern comes from only looking at this dts file.

Reply:
@anangl Good point. I have changed it accordingly.